scripts/addheaders.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+# Copyright 2023 IBM Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+if ! hash addlicense 2>/dev/null; then
+  go install github.com/google/addlicense@latest
```

Comment:
since we are running inside a Go module, will this install modify the go.mod / go.sum file?

Reply:
it shouldn't, at least I didn't spot anything being updated there.